src/scripts/main.js

```diff
@@ -1,3 +1,90 @@
 'use strict';
 
-// write code here
+const tHead = document.querySelector('thead');
+const tBody = document.querySelector('tbody');
+const rows = [...tBody.children];
+let reverse = false;
+
+function handleEvent(e) {
```

Comment:
```suggestion
function handleEvent(event) {
```
Let's use full, descriptive names